test-requirements.txt

```diff
@@ -6,3 +6,4 @@ pep8>=1.7.1
 pylint>=2.1.1
 testrepository>=0.0.20  # Apache-2.0/BSD
 pexpect==4.6.0
+PyYAML==3.13 # do not change this
```

Comment:
Don't need to add this to both 'requirements.txt' and 'test-requirements.txt'.

The run-time venv installs everything in 'requirements.txt' (https://github.com/open-power-ref-design-toolkit/power-up/blob/master/scripts/venv_install.sh#L24).

The tox venv installs everything in 'requirements.txt' _and_ 'test-requirements.txt' (https://github.com/open-power-ref-design-toolkit/power-up/blob/master/tox.ini#L12-L13).

Reply:
done